### ISSUE TITLE
[Question][config-ui] devlake admin background suddenly prompts [New Migration Scripts Detected]

### ISSUE THREAD

**Issue by SxjAccount** [apache/devlake#7992]
## Question
devlake has been in use for some time
devlake admin background suddenly prompts [New Migration Scripts Detected]

## Screenshots
![24-09-03_1](https://github.com/user-attachments/assets/c8d6bb75-83df-40f0-a7e8-fdbe5ad6a488)
![24-09-03_2](https://github.com/user-attachments/assets/167e5bcc-232e-40d5-af04-1e17cc404c4a)

## Additional context
After using Devlake for a while, it was able to enter the page normally before. Today, it suddenly appeared with 'New Migration Scripts Detected'. What is the reason for the appearance of 'New Migration Scripts Detected'. I have added some tables and data to the database generated by devlake. Will clicking on 'Proceed to Database Migration' on the screen cause changes to these tables and data?

**Comment by klesh** [comment]
The screenshot indicates an uptime of 4 hours, suggesting a recent restart. Have you made any changes to the database during this time? Perhaps modified records in the _devlake_migration_history table?

**Comment by SxjAccount** [comment]
> The screenshot indicates an uptime of 4 hours, suggesting a recent restart. Have you made any changes to the database during this time? Perhaps modified records in the _devlake_migration_history table?

My computer is set to turn on at 8 am and turn off at 6 PM every day.
![24-09-03_3](https://github.com/user-attachments/assets/a3386aad-b085-4667-8f1e-dd2d69ba6487)
That's my database up there, and now all of a sudden it's not working.

**Comment by SxjAccount** [comment]
![24-09-03_3](https://github.com/user-attachments/assets/7d94ef09-6257-4958-8eee-c5c262c91f8b)

**Comment by klesh** [comment]
How about other tables? any record?
Seems like the database is corrupted. 😂

**Comment by SxjAccount** [comment]
> How about other tables? any record? Seems like the database is corrupted. 😂

Do you know how the above situation came about? I didn't do anything with the server where devlake was installed, just set up an automatic shutdown.

**Comment by klesh** [comment]
No idea, it is likely caused by your server setup.

**Comment by klesh** [comment]
Try removing the automatic shutdown.

**Comment by SxjAccount** [comment]
> Try removing the automatic shutdown.

I deployed and set up EC2 automatic power on/off on AWS, but this situation suddenly occurred. I would like to observe the situation further and see if it is necessary to disable the automatic power on/off function

**Comment by klesh** [comment]
Absolutely! I'm genuinely curious about the outcome—please feel free to share it when you can. 🌟

**Comment by d4x1** [comment]
> > The screenshot indicates an uptime of 4 hours, suggesting a recent restart. Have you made any changes to the database during this time? Perhaps modified records in the _devlake_migration_history table?
> 
> My computer is set to turn on at 8 am and turn off at 6 PM every day. ![24-09-03_3](https://private-user-images.githubusercontent.com/85595603/363836951-a3386aad-b085-4667-8f1e-dd2d69ba6487.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY2NzMxODgsIm5iZiI6MTcyNjY3Mjg4OCwicGF0aCI6Ii84NTU5NTYwMy8zNjM4MzY5NTEtYTMzODZhYWQtYjA4NS00NjY3LThmMWUtZGQyZDY5YmE2NDg3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE4VDE1MjEyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkyMzkzMWNhNGIyN2YwZDJiNGUxZjc0OTEwNjk3ZjA3YThlYzMzYWNiZmE0NTgxOWIzOGI4NWE1ZDlmNmIyNTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.m3yLoqE-cYN1c8f5uQLirXTZlz_2tsdkgI_Td08bM-c) That's my database up there, and now all of a sudden it's not working.

@SxjAccount  From your screenshot,  lots of records have gone in table `_devlake_migration_history`.  I think it's the root cause. You can try to find why records are missing.

I am not an expert on MySQL but here are some possible reasons: 
1. There are many records but your db tool shows some of them only.
2. They are deleted by yourself/ some cronjobs/other people.
